### ISSUE TITLE
nordvpn cannot connect

### ISSUE THREAD

**Issue by curiousercreative** [pop-os/beta#385]
**How did you upgrade to 21.10? (Fresh install / Upgrade)**: Fresh
 

 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
```
$ apt policy nordvpn
nordvpn:
  Installed: 3.12.5
  Candidate: 3.12.5
  Version table:
 *** 3.12.5 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
        100 /var/lib/dpkg/status
     3.12.4 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.12.3 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.12.2 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.12.1-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.12.0-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.11.0-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.10.0-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.9.5-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.9.4-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.9.3-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.9.2-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.9.1-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.9.0-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.10 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.9 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.8 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.7 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.6 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.5 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.4 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.3 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.2 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.8.0 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.7.4 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.7.3 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.7.2 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.7.1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.7.0-3 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.7.0-2 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.6.1-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.6.0-5 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.6.0-4 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.6.0-3 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.6.0-2 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.6.0-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.5.0-2 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.5.0-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.4.0-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.3.1-3 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.3.1-2 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.3.1-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.3.0-4 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.3.0-3 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.3.0-2 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.3.0-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.3.0-0 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.2.0-4 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.2.0-3 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.2.0-2 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.2.0-1 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
     3.2.0-0 500
        500 https://repo.nordvpn.com/deb/nordvpn/debian stable/main amd64 Packages
```

**Issue/Bug Description:**
`nordvpn connect` fails

```
$ nordvpn connect
Connecting to United States #9216 (us9216.nordvpn.com)
Whoops! Connection failed. Please try again. If the problem persists, contact our customer support.
````

**Steps to reproduce (if you know):**

```bash
wget https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/nordvpn-release_1.0.0_all.deb
sudo dpkg -i nordvpn-release_1.0.0_all.deb
sudo apt update
sudo apt install nordvpn
sudo usermode -aG nordvpn $(whoami)

# reboot

nordvpn login
nordvpn connect
```

**Expected behavior:**
Successful connection as on latest Pop! 21.10

```bash
$ nordvpn connect
Connecting to United States #8671 (us8671.nordvpn.com)
You are connected to United States #8671 (us8671.nordvpn.com)!
```

**Comment by xploz1on** [comment]
I had some problems with ProtonVPN. The app would hang and not respond I gave up and was using ssh tunnel to a provate VPS. 
After a few days and some system updates, the ProtonVPN app works like it should, butter smooth. Have you tried the NordVPN app lately?

**Comment by curiousercreative** [comment]
@xploz1on thanks for the reply. I just tried updating and the result is the same

```bash
$ sudo apt update
$ sudo apt upgrade -y
$ sudo systemctl reboot
$ nordvpn connect
Connecting to United States #8671 (us8671.nordvpn.com)
Whoops! Connection failed. Please try again. If the problem persists, contact our customer support.
```

Actually, things got much worse after the updates were installed:
1. There's been a boot coup! My primary 21.10 install boot entry was overwritten and is bootable while my previous Beta boot entry is now unbootable. I'll have to manually recover my 21.10 boot entry :(
2. The system76-power menu no longer lists graphics modes for switching. I haven't looked into whether `nvidia-smi` lists the dGPU or whether the `system76-power` cli works for graphics mode switching...

**Comment by curiousercreative** [comment]
> There's been a boot coup! My primary 21.10 install boot entry was overwritten and is bootable while my previous Beta boot entry is now unbootable. I'll have to manually recover my 21.10 boot entry :(
Hmmm, I was able to easily restore 21.10 as my default boot entry (Pop_OS-current) by updating the UUID and EFI paths, but the beta boot entry looks the same as before (and similar to the working boot entry) but does not boot, just returns to the S76 logo bootscreen....
```bash
# cat /boot/efi/loader/entries/Pop_OS-beta.conf 
title Pop!_OS Beta
linux /EFI/Pop_OS-b14f1427-ee5e-455c-a54b-624481cdba28/vmlinuz.efi
initrd /EFI/Pop_OS-b14f1427-ee5e-455c-a54b-624481cdba28/initrd.img
options root=UUID=b14f1427-ee5e-455c-a54b-624481cdba28 ro quiet loglevel=0 systemd.show_status=false splash
```

UPDATE: after changing "systemd.show_status=false" to "systemd.show_status=true" the beta boot entry started working?!? Maybe just the act of modifying the boot entry had more to do with it?

>The system76-power menu no longer lists graphics modes for switching. I haven't looked into whether nvidia-smi lists the dGPU or whether the system76-power cli works for graphics mode switching...
`nvidia-smi` shows dGPU and 510 drivers but...
```bash
$ system76-power graphics
daemon returned an error message: "The name com.system76.PowerDaemon was not provided by any .service files"
```

I think I've seen this posted on reddit, so probably it's probably a known issue by now.

**Comment by Seibz** [comment]
Is there an update available for the application? As Ubuntu 22.04 is also launching this week, I suspect .deb releases of VPN managers will have an update to help with this. 

Regarding the system76-power daemon, please apply updates manually via your terminal and reboot to fix that.

`sudo apt update -y && sudo apt full-upgrade -y && sudo apt autoremove -y && sudo apt clean && flatpak update -y `

**Comment by curiousercreative** [comment]
Closing this after many updates from nordvpn. If someone can demonstrate this is still a problem, we should migrate this issue to the pop-os/pop repo.
